packages/compass-aggregations/src/modules/pipeline-builder/pipeline-parser/pipeline-parser.ts

```diff
@@ -239,7 +239,10 @@ export default class PipelineParser {
 }
 
 function getLineOnlySourceLocation(line: number) {
-  return { start: { line, column: 0 }, end: { line, column: 0 } };
+  return {
+    start: { line, column: 0 },
+    end: { line, column: 0 },
+  } as t.SourceLocation;
```

Comment:
This started failing when compiling compass-aggregations plugins, probably a version change somewhere that we merged recently that wasn't directly touching the package and so went unnoticed

Reply:
Saw the same change in the bson 6 mongosh 2 PR.

Reply:
Hmmm, true, probably I was jumping between branches and did `npm install` when doing changes to this bson 6 branch that affected this when I was testing it locally